package.json

```diff
@@ -20,7 +20,18 @@
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-react": "^7.22.0",
+    "husky": "=4",
```

Comment:
Is there a reason to avoid husky 5?

Reply:
See above comment

Reply:
Also fine with upgrading to v5 if we want, just haven't taken the time to investigate the new setup to determine how to automatically install the hooks for any new contributors.

Reply:
I think we can separate this into a separate issue (and PR in the future) since it's not a big deal and contributors are probably more familiar with v4 at the moment